src/containers/Admin/AdminContainter.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from "react";
+import { authAccessToken } from "../../utils/auth";
+import { AdminLogin } from "../../components/Admin/AdminLogin/AdminLogin";
+import { AdminPopUpClinic } from "../../components/Admin/AddPopUpClinic/AdminPopUpClinic";
+
+export function AdminContainer() {
+  const [accessToken, setAccessToken] = useState(
+    window.sessionStorage.getItem("bearerToken") || null,
+  );
+
+  if (accessToken && !authAccessToken(accessToken)) {
+    window.sessionStorage.removeItem("bearerToken");
+    setAccessToken(null);
```

Comment:
You may want to add `return <></>;` after this line. `setAccessToken` is an async operation, so there could be a brief render where `<AdminPopUpClinic />` is displayed.